atomium/includes/registry.inc

```diff
@@ -15,7 +15,7 @@
  * @see atomium_theme()
  * @see hook_theme()
  */
-function _atomium_theme(&$existing, $type, $theme, $path) {
+function _atomium_theme(array &$existing, $type, $theme, $path) {
```

Comment:
Accordingly, to what we already discussed, it needs to change the parameters, it should not be passed by reference as it is not to be overridden. please check  [here](https://github.com/drupal/drupal/blob/7.x/includes/theme.inc#L534)

Reply:
Thanks, fixed!